### PR TITLE
[Feature Test] Test Double click feature on canvas

### PR DESCRIPTION
**Pull request by Abhishek-kumar09**
**Description**
Restart of the Meshery Extension Test suite and feature test

- Add support and TS files for commands
- Add test for canvas double click operation

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Abhishek-kumar09** [comment]
Tests would be failing here, because it is running over release rather than any branch on meshery-extension

**Comment by MarioArriaga92** [comment]
Taking a look

**Comment by MarioArriaga92** [comment]
> Tests would be failing here, because it is running over release rather than any branch on meshery-extension

@Abhishek-kumar09 but when a meshery-extension PR runs its checks it would use the source branch to run tests, right? (in order to find regression issues as earliest as possible of course)

File: cypress/e2e/e2e/v2.design_spec.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+// type definitions for Cypress object "cy"
+/// <reference types="cypress" />
+/// <reference types="../../support" /> 
+
+import { expect } from "chai";
```
Comment (MarioArriaga92) [inline cypress/e2e/e2e/v2.design_spec.js:5]: @Abhishek-kumar09 chai is already bundled with cypress so its not necessary to import it nor do npm install at all: https://docs.cypress.io/guides/references/bundled-libraries#Chai

**Review by MarioArriaga92 — APPROVED** [review]
LGTM just remove the unnecessary "chai" import (see code comment)

File: cypress/e2e/e2e/v2.design_spec.js (hunk 1 above)
Comment (Abhishek-kumar09) [reply cypress/e2e/e2e/v2.design_spec.js:5]: Oh, mario, I was getting import problems during running this test, that's why. 
But yeah, I would take this PR https://github.com/layer5labs/meshery-extensions-packages/pull/85 as a motivation to wrap the assertions under cypress object

**Comment by Abhishek-kumar09** [comment]
> @Abhishek-kumar09 but when a meshery-extension PR runs its checks it would use the source branch to run tests, right? (in order to find regression issues as earliest as possible of course)

Yes

File: cypress/e2e/e2e/v2.design_spec.js (hunk 1 above)
Comment (Abhishek-kumar09) [reply cypress/e2e/e2e/v2.design_spec.js]: Seems fine now. Thanks for pointing out

**Comment by Abhishek-kumar09** [comment]
All Checks Passed ✅

**Pull request merged**